packaging/dockerfiles/Dockerfile.ubuntu24.04

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:24.04
```

Comment:
IMHO we could factor out the Dockerfiles per distribution: one for Ubuntu, one for Rocky; the version number could be parameterized from the GitHub workflows; see e.g. https://github.com/KhiopsML/khiops-python/blob/dev/packaging/docker/khiopspydev/Dockerfile.ubuntu for the Dockerfile and https://github.com/KhiopsML/khiops-python/blob/dev/.github/workflows/dev-docker.yml for their instantiations.

Reply:
I disagree, it may be some specificities on each distro. For instance, on ubuntu 20.04 we set a specific version of cmake.

Reply:
OK, I see your point about this specific case; however, for such cases, I wonder whether a base `Dockerfile.ubuntu` image could be an initial Docker image that the various particular versions would derive `FROM` and add specific layers.